specification/attestation/Attestation/routes.tsp

```diff
@@ -0,0 +1,272 @@
+    TpmAttestationRequest,
+    TpmAttestationResponse
+  >;
+
+  /**
+   * Processes a SEV SNP Boot chain. The type of artifact produced is dependent upon
+   * attestation policy.
+   */
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Existing API - must use current operation signature for backward compatibility"
+  @summary("Attest to an SEV SNP Virtual Machine.")
+  @route("/attest/SevSnpVm")
+  @post
+  attestSevSnpVm is Azure.Core.Foundations.Operation<
+    AttestSevSnpVmRequest,
+    AttestationResponse
+  >;
+
+  /**
+   * Processes an TDX quote, producing an artifact. The type of artifact produced is
+   * dependent upon attestation policy.
+   */
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Existing API - must use current operation signature for backward compatibility"
+  @summary("Attest to a TDX Virtual Machine.")
+  @route("/attest/TdxVm")
+  @post
+  attestTdxVm is Azure.Core.Foundations.Operation<
+    AttestTdxVmRequest,
+    AttestationResponse
+  >;
+}
+
+interface TcbBaselinesOperationGroup {
+  /**
+   * Retrieves a list of all Azure supported baseline details for the a TEE type
+   * along with an indication of which one is in enforced for the attestation
+   * provider.
+   */
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Existing API - must use current operation signature for backward compatibility"
+  @summary("""
+    Retrieves a list of all Azure supported baseline details for the a TEE type
+    along with an indication of which one is in enforced for the attestation
+    provider.
+    """)
+  @route("/tcbbaselines/{attestationType}")
+  @get
+  get is Azure.Core.Foundations.Operation<
+    {
+      /**
+       * Specifies the trusted execution environment to be used to validate the evidence
+       */
+      @path
+      attestationType: AttestationType;
+    },
+    TcbBaselineResult
+  >;
+}
+
+interface SigningCertificatesOperationGroup {
+  /**
+   * Retrieves metadata signing certificates in use by the attestation service
+   */
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Existing API - must use current operation signature for backward compatibility"
+  @summary("Retrieves the attestation signing keys in use by the attestation service")
+  @route("/certs")
+  @get
+  get is Azure.Core.Foundations.Operation<
+    {},
+    JsonWebKeySet & {
+      @header
+      contentType: SigningCertificatesContentType;
+    }
+  >;
+}
+
+interface MetadataConfigurationOperationGroup {
+  /**
+   * Retrieves metadata about the attestation signing keys in use by the attestation
+   * service
+   */
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Existing API - must use current operation signature for backward compatibility"
+  @summary("Retrieves the OpenID Configuration data for the Azure Attestation Service")
+  @route("/.well-known/openid-configuration")
```

Comment:
Is the `.` actually in the path?

Reply:
based on this docs it is as part of openid

https://openid.net/specs/openid-connect-discovery-1_0.html#toc


```html
GET /.well-known/openid-configuration HTTP/1.1
Host: example.com
```